packages/fuels-accounts/src/provider.rs

```diff
@@ -98,6 +96,7 @@ impl ResourceQueries {
     }
 }
 
+#[derive(Default)]
```

Comment:
Maybe this is not the behaviour we want. Using default for AssetId might be different than the asset id configured in consensus parameters.
My expectation would be that the default always filters for the base asset.

Reply:
Also the default for asset id is going to go behind a "test-only" feature over in fuel-* crates. 
Maybe have it be an `Option` and we'll resolve it to the base asset later when querying? If we care that much about having a default.  We probably do, it seems that the ux would take a hit without being able to say `..Default::default()`